docs/general/aws-eks-iam-for-s3.md

```diff
@@ -0,0 +1,81 @@
+# Using IAM for Service Account to Configure the S3 State Store Backend
+
+## Overview
+
+This document describes how to use IAM for Service Account to configure the S3 state store backend.
+
+## Prerequisites
+
+1. The RisingWave should be running on the AWS EKS cluster.
+
+## Prepare
+
+1. Follow the instructions [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) to create
+   a S3 bucket.
+2. Follow the instructions [here](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
+   to create an IAM role for service account. These two steps must be completed:
+    1. [Creating an IAM OIDC provider for your cluster](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+    2. [Configuring a Kubernetes service account to assume an IAM role](https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html)
+3. Attach the IAM role
+   with [AmazonS3FullAccess](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonS3FullAccess.html).
+    1. You can customize the IAM policy to limit the access to the S3 bucket.
```

Comment:
Consider link this as an example: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_resource.html

Reply:
Sounds good.